### PR TITLE
feat: Adds kraken GCP rights

### PR DESCRIPTION
**Pull request by tfelix**
Adds additional rights for GCP tenant billing.

### CODE REVIEW

File: docs/meshstack.gcp.index.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ We recommend using [externally provisioned identities](./meshstack.identity-fede
 
 Operators need to setup a GCP Organization to be used by meshStack. Please review the official GCP documentation on [creating and managing organizations](https://cloud.google.com/resource-manager/docs/creating-managing-organization).
 
-### meshfed-service IAM Role
+### Service Account Configuration
 
 meshStack needs a well-defined set of permissions for its automation. meshStack is designed so that it **does not require
 access to workload**. We highly recommend that permissions are configured according to the principle of least privilege.
```
Comment (chewbeccy) [inline docs/meshstack.gcp.index.md:34]:
```suggestion
access to workload**. We highly recommend that permissions are configured according to the "least privilege" principle.
```

File: docs/meshstack.gcp.metering.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ roles/bigquery.jobUser
 roles/bigquery.dataViewer
 ```
 
+If you want to use the billing module of meshStack you also need another service principal, which is able to discover and list projects. We recommend to also create its own IAM role and assign it to the service account together with the following permissions:
```
Comment (chewbeccy) [inline docs/meshstack.gcp.metering.md:19]:
```suggestion
To enable the billing module of meshStack a service principal, which is able to discover and list projects, needs to be defined. We recommend to also create its own IAM role and assign it to the service account together with the following permissions:
```

**Review by chewbeccy — APPROVED** [review]
Made some suggestions. We should not address users directly with "you". But to be honest I dont know if this is currently already the case in some parts of the documentation.

**Comment by tfelix** [comment]
probably not but is fine with me.

**Pull request merged**